src/main/groovy/org/shipkit/gradle/ReleaseConfiguration.java

```diff
@@ -119,6 +121,22 @@ public String getPreviousReleaseVersion() {
     public class GitHub {
 
         /**
+         * GitHub API endpoint address, for example:  https://api.github.com/
+         */
+        public String getApiUrl() {
+            return getString("gitHub.apiUrl");
+        }
+
+        /**
+         * See {@link #getApiUrl()}
+         *
+         * @param apiUrl GitHub (or gitHub enterprise) API endpoint address, for example: https://api.github.com/
+         */
```

Comment:
This "/" at the end of example URL is important. In GitHubProjectContributorsBuilder you concatenate "/repos" to apiUrl so it has to not end with slash. 
See:
https://api.github.com//repos/mockito/mockito/contributors?access_token=e7fe8fcdd6ffed5c38498c4c79b2a68e6f6ed1bb

This one doesn't work. 

At first we can include information that URL should not be ended with slash, but it would be good to have a code that handles URL with slash and without it.